docs/fastdds/faq/dds_layer/dds_layer.rst

```diff
@@ -0,0 +1,750 @@
+.. include:: ../../../03-exports/aliases.include
+.. include:: ../../../03-exports/aliases-api.include
+.. include:: ../../../03-exports/roles.include
+
+.. _freq_dds_layer_questions:
+
+DDS LAYER Frequently Asked Questions
+====================================
+
+CORE
+----
+
+Entity
+^^^^^^
+
+.. collapse::  What is the significance of a unique ID in the context of DDS and RTPS entities, and why is it important to have a shared ID between these entities?
+
+
+
+
+    |br|
+
+    The unique ID ensures that each entity within the distributed system can be distinctly identified. This is crucial for maintaining the integrity and consistency of communications and operations within the system. By having a shared ID between DDS (Data Distribution Service) and RTPS (Real-Time Publish-Subscribe) entities, the system can seamlessly map and correlate data across different communication protocols and frameworks. For further information go to :ref:`dds_layer_core_entity`.
+
+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
```

Comment:
I am curious about why these lines need to be that long, and why the linter does not complain (max 120 characters per line).
Can you confirm that the changes introduced in this PR are being correctly tested?